docker-compose.yml

```diff
@@ -40,6 +43,9 @@ services:
     - logvolume:/var/log
     - couchdbvolume:/couchdb/data
     environment:
+      DEBUG_COLORS: "true"
+      TERM: xterm-256color
+      COLORTERM: truecolor
```

Comment:
rec only using these setting in this openemr docker (would remove them from the others (couchdb, ldap, etc) since it is not working on those)

Reply:
@bradymiller it doesn't seem to work on a lot of docker which is infuriating

Reply:
yep, guessing one of the developer command line packages openemr brings in luckily supports the color thing, which is really nice btw!